app/directives/pdf/pdf.directive.js

```diff
@@ -41,19 +41,21 @@ angular.module('dashboard')
                     element.addClass('db-visible-pdf');
                 }
 
+                function composeContent(data) {
+                    element.empty();
+                    var content = '<iframe src="' + noContentUri + '"></iframe>';
+                    if (angular.isObject(data) && angular.isString(data.uri) && data.uri.length) {
+                        content = '<iframe src="' + data.uri + paramSeparator(data.uri) + params + '"></iframe>';
+                    }
+                    element.append($compile(content)(scope));
+                    show();
+                }
+
                 scope.$watch(
                     function () {
                         return { uri: scope.uri };
                     },
-                    function (data) {
-                        element.empty();
-                        var content = '<iframe src="' + noContentUri + '"></iframe>';
-                        if (angular.isObject(data) && angular.isString(data.uri) && data.uri.length) {
-                            content = '<iframe src="' + data.uri + paramSeparator(data.uri) + params + '"></iframe>';
-                        }
-                        element.append($compile(content)(scope));
-                        show();
-                    },
+                    composeContent,
```

Comment:
On mobile attachments are not working -> Could not resolve 'app.signitem.signattlist' from state 'app.sign'

LGTM with comments